charts/agent-control/templates/install-flux.yaml

```diff
@@ -0,0 +1,46 @@
+{{- if index .Values "agent-control-cd" }}
+{{- if (index .Values "agent-control-cd" "flux2").enabled -}}
+{{- $installJobName := include "newrelic.common.naming.truncateToDNSWithSuffix" ( dict "name" (include "newrelic.common.naming.fullname" .) "suffix" "flux-install-job" ) -}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  annotations:
+    helm.sh/hook: post-install,post-upgrade
+    helm.sh/hook-delete-policy: before-hook-creation,hook-succeeded
+    helm.sh/hook-weight: "20"
+  name: {{ $installJobName }}
+  namespace: {{ .Release.Namespace }}
+spec:
+  backoffLimit: 2
+  template:
+    spec:
+      restartPolicy: Never
+      serviceAccountName: {{ include "newrelic.common.naming.truncateToDNSWithSuffix" (dict "name" .Release.Name "suffix" "flux-install-job") }}
+      containers:
+        - name: install-flux
+          image: alpine/helm:3.17.1
+          resources:
+            limits:
+              cpu: 100m
+              memory: 128Mi
+            requests:
+              cpu: 50m
+              memory: 64Mi
+          command: ["/bin/sh", "-c"]
+          args:
+            - |
+              helm repo add newrelic https://helm-charts.newrelic.com
```

Comment:
Do we need to consider custom internal repositories? (https://helm-charts.newrelic.com might not be available for particular environments)

Reply:
Done in last commit.